index.css

```diff
@@ -0,0 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+html, body {
+    @apply overflow-hidden;
+}
```

Comment:
Dette er vel html-tags som bare brukes en gang, så da kan man sikkert bare skrive klassen rett på de i html-markupen for å være mer "tailwindish".

Reply:
Får warnings på at className ikke er tillatt på slike elementer. `head` og `body` er nokk litt spesielle sånn sett.

Reply:
Det brukes vel i index.html så da må det isåfall være class og ikke className

Reply:
Det var egentlig ikke så nøye, bare flisespikkeri